CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 -   Added 1rem default padding to `<blui-empty-state>` ([#336](https://github.com/brightlayer-ui/angular-component-library/issues/336)).
 
+### Fixed
+
+-   Fixed `<blui-app-bar>` not listening for new scroll elements `onChanges`.
```

Comment:
Can you just add a link to the issue, like the entry above, before you merge this?